### PR TITLE
Add support for `logit_bias` and improve documentation

### PR DESCRIPTION
**Pull request by julianschiavo**
See [OpenAI documentation](https://platform.openai.com/docs/api-reference/chat/create#chat/create-logit_bias) on `logit_bias`:

> Modify the likelihood of specified tokens appearing in the completion.

> Accepts a json object that maps tokens (specified by their token ID in the tokenizer) to an associated bias value from -100 to 100. Mathematically, the bias is added to the logits generated by the model prior to sampling. The exact effect will vary per model, but values between -1 and 1 should decrease or increase likelihood of selection; values like -100 or 100 should result in a ban or exclusive selection of the relevant token.

### CODE REVIEW

**Comment by adamrushy** [comment]
Nice improvements 👏🏼 appreciate the contribution!

**Comment by julianschiavo** [comment]
Happy to help, thank you for the library! It's been super useful.
